.gitignore

```diff
@@ -0,0 +1,10 @@
+.buildlog
```

Comment:
Less is more here: https://dart.dev/guides/libraries/private-files

Reply:
I just copied this from some other repo. :)

I'll just go wtih
* dart-tool/
* .packages
* pubspec.lock

I'm not "build"-ing anything.